### PR TITLE
Enable edge-to-edge

### PR DESCRIPTION
**Pull request by zsmb13**

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/detail/DetailScreen.kt [hunk 1]
```diff
@@ -83,7 +86,9 @@ private fun ObjectDetails(
                 }
             }
         },
-        modifier = modifier,
+        modifier = modifier.padding(
+            WindowInsets.systemBars.asPaddingValues()
+        ),
```
Comment (terrakok) [inline composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/detail/DetailScreen.kt:91]: .windowInsetsPadding(WindowInsets.systemBars)

File: composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt [hunk 2]
```diff
@@ -68,7 +72,9 @@ private fun ObjectGrid(
     LazyVerticalGrid(
         columns = GridCells.Adaptive(180.dp),
         modifier = modifier.fillMaxSize(),
-        contentPadding = PaddingValues(8.dp)
+        contentPadding = WindowInsets.systemBars
+            .add(WindowInsets(left = 8.dp, right = 8.dp))
+            .asPaddingValues()
```
Comment (terrakok) [inline composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt:77]: WindowInsets.safeDrawing.asPaddingValues()

Comment (terrakok) [reply composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt:77]: Your code:
image

My code:
image

File: composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/detail/DetailScreen.kt (hunk 1 above)
Comment (terrakok) [inline composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/detail/DetailScreen.kt:91]: .windowInsetsPadding(WindowInsets.systemBars)

**Review by terrakok — CHANGES_REQUESTED** [review]

File: composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt (hunk 2 above)
Comment (zsmb13) [reply composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt:77]: I have definitely not looked at landscape, my bad. But even with this code, I often get the content's right edge clipped both on a real Pixel device with a cutout and on an emulator.

image

Comment (zsmb13) [reply composeApp/src/commonMain/kotlin/com/jetbrains/kmpapp/screens/list/ListScreen.kt:77]: Or, rotated the other way, content is drawn below the cutout still

image

**Review by terrakok — APPROVED** [review]

**Pull request merged**